src/mame/includes/tetrisp2.h

```diff
@@ -6,6 +6,7 @@
 #include "video/ms32_sprite.h"
 #include "emupal.h"
 #include "tilemap.h"
+#include "sound/ymz280b.h"
```

Comment:
Please put this with the other headers from src/devices, not after the headers from src/emu.